controllers/storagecluster/cephcluster.go

```diff
@@ -405,6 +405,13 @@ func getReplicasPerFailureDomain(sc *ocsv1.StorageCluster) int {
 	return defaults.ReplicasPerFailureDomain
 }
 
+func getReplicaSize(sc *ocsv1.StorageCluster) uint {
```

Comment:
Rename this to something like `getCephPoolReplicatedSize()`, and please add a quick comment on what this function is for.

Reply:
done